cpp/tests/utilities/scalar_wrapper.cuh

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2018, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SCALAR_WRAPPER_H
+#define SCALAR_WRAPPER_H
+
+#include "cudf.h"
+#include "cudf_test_utils.cuh"
+#include "rmm/rmm.h"
+#include "utilities/bit_util.cuh"
+#include "utilities/type_dispatcher.hpp"
+
+#include <thrust/equal.h>
+#include <thrust/logical.h>
+#include <bitset>
+
+namespace cudf {
+namespace test {
+
+/**---------------------------------------------------------------------------*
+ * @brief Wrapper for a gdf_scalar used for unit testing.
+ *
+ * An abstraction on top of a gdf_scalar that provides functionality for
+ * allocating, intiailizing, and otherwise managing gdf_scalar's for passing to
```

Comment:
```suggestion
 * allocating, intiailizing, and otherwise managing gdf_scalars for passing to
```

Reply:
[You're right](https://www.grammarbook.com/punctuation/apostro.asp). See Rule 2b